requirements/dev.txt

```diff
@@ -21,13 +23,22 @@ click-log==0.3.2
 click==7.1.2
     # via
     #   click-log
+    #   code-annotations
     #   edx-lint
     #   pip-tools
+code-annotations==1.0.2
+    # via edx-lint
+colorama==0.4.4
+    # via twine
 distlib==0.3.1
     # via virtualenv
+django==3.1.5
```

Comment:
I don't think we want to upgrade past 2.2.x in dev environments yet.  And actually, can we avoid this Django dependency in code-annotations and edx-lint?

Reply:
Added constraint `Django<2.3` in `constraints.txt` file to keep the LTS release.

Reply:
Both `edx-lint` and `code-annotations` have `django` in the `base.in` dependencies and have the constraint `django<2.3` either in `constraints.txt` or in `common constraints`, so adding the constraint in this repo was the only way I found to contain the `django` version to the required threshold.

Reply:
There will probably be a future ticket to make those dependencies optional, but this can go ahead in the meantime.  I'd just rather not jump into the Django 3.x upgrades until we've made sure that we aren't requiring Django in repos that don't actually need it.